components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/utils/RESTAPIAdminClient.java

```diff
@@ -26,71 +26,77 @@
 
 public class RESTAPIAdminClient extends AbstractAPIGatewayAdminClient {
 
-    private RestApiAdminStub restApiAdminStub;
-    private String qualifiedName;
-    private Environment environment;
-    
-    public RESTAPIAdminClient(APIIdentifier apiId, Environment environment) throws AxisFault {
-        this.qualifiedName = apiId.getProviderName() + "--" + apiId.getApiName() + ":v" + apiId.getVersion();
-        String providerDomain = apiId.getProviderName();
-        providerDomain=providerDomain.replace("-AT-", "@");
-        restApiAdminStub = new RestApiAdminStub(null, environment.getServerURL() + "RestApiAdmin");
-        setup(restApiAdminStub, environment);
-        this.environment = environment;
-    }
-    
+	private RestApiAdminStub restApiAdminStub;
```

Comment:
This code snippet wasn't added, only formatted.
Variable `restApiAdminStub` wasn't renamed because all methods in the `org.wso2.carbon.rest.api.stub.RestApiAdminStub` class has the "api" infix (e.g. see line 56). IMO we should refactor the whole `RestApiAdminStub` class to reflect apps operations instead of APIs.